### PR TITLE
chore(i18n): enrich HTTP API desc

### PR DESCRIPTION
**Pull request by HJianBo**

### CODE REVIEW

File: apps/emqx_management/i18n/emqx_mgmt_api_banned_i18n.conf [hunk 1]
```diff
@@ -2,112 +2,97 @@ emqx_mgmt_api_banned {
...
-            zh: """黑名单创建者"""
+            en: """Ban Initiator"""
+            zh: """封禁发起者"""
         }
     }
     reason {
         desc {
-            en: """Banned reason"""
-            zh: """黑名单创建原因"""
+            en: """Banned reason, record the reason why the current object was banned."""
```
Comment (thalesmg) [inline apps/emqx_management/i18n/emqx_mgmt_api_banned_i18n.conf:70]:
```suggestion
            en: """Ban reason, record the reason why the current object was banned."""
```

File: apps/emqx_gateway/src/emqx_gateway_api_listeners.erl [hunk 2]
```diff
@@ -358,7 +358,9 @@ schema("/gateways/:name/listeners") ->
         'operationId' => listeners,
         get =>
             #{
+                tags => [<<"Gateways">>],
                 desc => ?DESC(list_listeners),
+                summary => <<"List All Listener">>,
```
Comment (thalesmg) [inline apps/emqx_gateway/src/emqx_gateway_api_listeners.erl:363]:
```suggestion
                summary => <<"List All Listeners">>,
```

File: apps/emqx_gateway/i18n/emqx_gateway_api_listeners_i18n.conf [hunk 3]
```diff
@@ -2,105 +2,109 @@ emqx_gateway_api_listeners {
...
+            zh: """获取监听器的认证器配置。"""
         }
     }
 
     add_listener_authn {
         desc {
-            en: """Add authentication for the listener"""
-            zh: """为指定监听器添加认证器"""
+            en: """Enable authenticator for specified listener for client authentication.</br>
+When authenticator is enabled for a listener, all clients connected to that listener will use that authenticator for authentication."""
```
Comment (thalesmg) [inline apps/emqx_gateway/i18n/emqx_gateway_api_listeners_i18n.conf:50]:
```suggestion
When authenticator is enabled for a listener, all clients connecting to that listener will use that authenticator for authentication."""
```

File: apps/emqx_management/i18n/emqx_mgmt_api_banned_i18n.conf [hunk 4]
```diff
@@ -2,112 +2,97 @@ emqx_mgmt_api_banned {
...
-
-    create_banned {
-        desc {
-            en: """List banned."""
-            zh: """列出黑名单"""
-        }
-        label {
-            en: """List Banned"""
-            zh: """列出黑名单"""
+            en: """The banned object was not found in the blacklist."""
```
[the remark this answers is not in the document]
Comment (HJianBo) [reply apps/emqx_management/i18n/emqx_mgmt_api_banned_i18n.conf:33]: 🤔 I'm not sure what's better. what do you think? @zmstone

[the remark this answers is not in the document]
Comment (lafirest) [reply apps/emqx_management/i18n/emqx_mgmt_api_banned_i18n.conf:33]: the `blacklist` is a traditional word, but I think `blocklist` may be better today, or `banned list` ?

File: apps/emqx_authn/src/emqx_authn_api.erl [hunk 5]
```diff
@@ -34,7 +34,7 @@
 % Swagger
 
 -define(API_TAGS_GLOBAL, [<<"Authentication">>]).
--define(API_TAGS_SINGLE, [<<"Listener authentication">>]).
```
Comment (lafirest) [inline apps/emqx_authn/src/emqx_authn_api.erl:37]: I'm curious why do this, in our docs, the format of the title is `Foo Boo`, not `Foo boo`

**Review by ieQu1 — APPROVED** [review]

**Pull request merged**